docs/en/ingest-management/elastic-agent/elastic-agent-capabilities.asciidoc

```diff
@@ -24,13 +24,16 @@ Let's use the following configuration and the capabilities defined above.
 [source,yaml]
 ----
 inputs:
-	- type: logfile
```

Comment:
I'm not sure capabilities are actually linked from anywhere, so this might not be published. I'd have to check if this was actually a GA'd feature, it might not be.